### DISCUSSION TITLE
Trigger matching before typing the whole trigger.

### DISCUSSION THREAD

**Discussion by jamesgraham320** [sodenn/lexical-beautiful-mentions#761] [Q&A]
Is there a way to override/adjust how the plugin determines when to show a trigger's items? If i have longer trigger e.g. "files" what would be the best way to have the menu pop up as soon as the user starts typing the word files. E.x. "f" -> menu opens and stays open as they continue to type the word "files". Something like implementing a partialTriggerMatch function. Thanks!

**Comment by sodenn** [comment]
Hi @jamesgraham320. Currently, the plugin doesn’t support overriding how triggers determine when to show items. As an alternative, you can use the `combobox` prop, which displays all available triggers. This is especially helpful for longer or unknown triggers.
